tests_v2/test_ssh_remote_image_drop_upload.py

```
@@ -0,0 +1,247 @@
+
+
+SOCKET_PATH = os.environ.get("CMUX_SOCKET_PATH", "/tmp/cmux-debug.sock")
+
+
+def _must(condition: bool, message: str) -> None:
+    if not condition:
+        raise cmuxError(message)
+
+
+def _run(cmd: list[str], *, check: bool = True) -> subprocess.CompletedProcess[str]:
+    proc = subprocess.run(cmd, capture_output=True, text=True, check=False)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)